.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+    -   repo: https://github.com/realm/SwiftLint
+        rev: 0.50.3
```

Comment:
Quick note here @henrikth93: pre-commit revs are directly from the projects that they're leveraging, so it's good to always check what the latest version is! I figured this out recently when setting pre-commit up for multiple repos 😇 For this one it's actually at 0.55.1. Have it updated locally and will send it along shortly!